lib/Importer.js

```diff
@@ -401,6 +401,11 @@ export default class Importer {
       return undefined;
     }
 
+    const distanceToCurrentFile = (importPath: string): number => {
+      const match = path.relative(this.pathToCurrentFile, importPath).match(/\//g);
```

Comment:
That sounds right... should we check for that?

Now that I think about it, I should probably check for path.sep instead of / to handle the windows case? Or are these paths normalized already?